client/commonFramework/end.js

```diff
@@ -164,7 +164,7 @@ $(document).ready(function() {
             return common.updateOutputDisplay('' + err);
           }
           common.codeStorage.updateStorage(challengeName, originalCode);
-          common.displayTestResults(tests);
+          common.displayTestResults(tests, true);
```

Comment:
changing this to `common.displayTestResults(tests, false);` seems to fix the issue.

You had only disabled the scroll on load for HTML challenges.